cmd/juju/commands/debuglog.go

```diff
@@ -163,16 +163,16 @@ func (c *debugLogCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.Var(cmd.NewAppendStringsValue(&c.params.IncludeLabel), "include-label", "Only show log messages for these logging labels")
 	f.Var(cmd.NewAppendStringsValue(&c.params.ExcludeLabel), "exclude-label", "Do not show log messages for these logging labels")
 
-	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]")
+	f.StringVar(&c.level, "l", "", "Log level to show, one of [TRACE, DEBUG, INFO, WARNING, ERROR]. Logs of higher severity (e.g. ERROR) will still be shown when a lower level (e.g. INFO) is selected")
 	f.StringVar(&c.level, "level", "", "")
 
-	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines, and continue to append")
+	f.UintVar(&c.params.Backlog, "n", defaultLineCount, "Show this many of the most recent (possibly filtered) lines and continue to append new ones")
 	f.UintVar(&c.params.Backlog, "lines", defaultLineCount, "")
-	f.UintVar(&c.params.Limit, "limit", 0, "Exit once this many of the most recent (possibly filtered) lines are shown")
-	f.BoolVar(&c.params.Replay, "replay", false, "Show the entire (possibly filtered) log and continue to append")
+	f.UintVar(&c.params.Limit, "limit", 0, "Show this many of the most recent (possibly filtered) lines and then exits")
```

Comment:
```suggestion
	f.UintVar(&c.params.Limit, "limit", 0, "Show this many of the most recent (possibly filtered) lines and then exit")
```

I'd dropped "possibly filtered", it's not saying enough or it's saying too much.

Reply:
I agree, was unclear to me when I first saw the description, thanks for the suggestion Simon :)